tests/CatalogService.IntegrationTest/Usings.cs

```diff
@@ -0,0 +1 @@
+global using NUnit.Framework;
```

Comment:
For this case, I used NUnit because I thought it would be simpler to have the setup class which is repsonsible for creating the database structure for the tests. In xUnit it is possible to have that setup and teardown features but it would require the creation of more classes. So that's why I chose NUnit.